source/index-management.rst

```diff
@@ -669,6 +669,13 @@ Common API Components
      * filter expression are included in the index.
      */
     partialFilterExpression: Document;
+
+    /**
+     * Optionally specifies a collation to use for the index. If not specified, the default collation
```

Comment:
I'm reading in the spec document that, upon index creation:

The absence of the collation parameter means that the index will use the collection’s collation. The system will automatically populate the index catalog’s persistent metadata infoObj with the inherited collation.

@dstorch can you confirm that an index created on a collection will use the collection's default collation in mongod, if it has one?